modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -46,7 +46,7 @@ It checks if the user is already logged in on Bonita
 ====
 
 Bonita `username` should match the `prefered_username` or one claims returned by the OIDC provider in the JWT token in the response.
```

Comment:
I dont get this phrase....

Reply:
Would need your input on this one as well @abirembaut

Reply:
```suggestion
The `username` attribute of Bonita user accounts should match one of the _claims_ of the JWT token returned in the OIDC provider response (by default the the `prefered_username` is used but you can change the configuration to use another claim).
```
is it clearer like that ? I think so.

Reply:
yes :) thank you!